com.endlessm.MultiEknServices.json.in

```diff
@@ -0,0 +1,84 @@
+{
+    "app-id": "com.endlessm.MultiEknServices",
```

Comment:
This is a really internal-facing name. To understand it, you have to know the history of EknServices. Can you think of a different one?

Reply:
I did wonder about just delivering this as an upgrade to EknServices, then we would rebuild the constituent EknServices1 under its own (internal) name.